shaders/src/common_shading.fs

```diff
@@ -3,7 +3,8 @@
 highp mat3  shading_tangentToWorld;   // TBN matrix
 highp vec3  shading_position;         // position of the fragment in world space
       vec3  shading_view;             // normalized vector from the fragment to the eye
-      vec3  shading_normal;           // normalized normal, in world space
+      vec3  shading_normal;           // normalized transformed normal, in world space
+      vec3  shading_geometricNormal;  // normalized geometric normal, in world space
```

Comment:
Maybe add "(before bump mapping)" to the comment for geometric normal, since the distinction could be confusing to newcomers.